packages/flutter/lib/src/material/slider.dart

```diff
@@ -2018,25 +2072,26 @@ class _SliderDefaultsM3 extends SliderThemeData {
   Color? get disabledInactiveTrackColor => _colors.onSurface.withOpacity(0.12);
 
   @override
-  Color? get disabledSecondaryActiveTrackColor => _colors.onSurface.withOpacity(0.12);
+  Color? get disabledSecondaryActiveTrackColor => _colors.onSurface.withOpacity(0.38);
 
   @override
-  Color? get activeTickMarkColor => _colors.onPrimary.withOpacity(0.38);
+  Color? get activeTickMarkColor => _colors.secondaryContainer;
 
   @override
-  Color? get inactiveTickMarkColor => _colors.onSurfaceVariant.withOpacity(0.38);
+  // TODO(tahatesser): Update this hard-coded value to use the correct token value.
```

Comment:
Can you file an issue to follow up and include it here?

Reply:
Filed https://github.com/flutter/flutter/issues/153271, will include it here